src/workloads/execution/CreateIndex.yml

```diff
@@ -0,0 +1,172 @@
+# This workload has 2 actors, InsertData and IndexCollection. InsertData inserts documents containing all types of indexes, and IndexCollection creates indexes for each of them, one at a time.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 150000
+    BatchSize: 10000
+    Document:
+      loc2d: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
+      loc2dSphere: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}]
+      point: { type: "Point", coordinates: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}] }
+      randomString: {^RandomString: {length: 50}}
+      randomInt: {^RandomInt: {min: -10000000, max: 10000000}}
+      "attributes": { randomString : randomString, randomString : randomString, randomString : { randomString :  randomInt, randomString :  randomInt }, randomString : { randomString : randomString }}
```

Comment:
I don't think this does what you think it does. You probably need to do the value randomization like above.

Reply:
FYI: you cannot generate a random "key" value, only a random "value" value. It looks like you want to generate random keys here. I double checked with @guoyr on generating keys.